exporting/read_config.c

```diff
@@ -391,7 +391,10 @@ struct engine *read_exporting_config()
         else
             tmp_instance->config.options &= ~EXPORTING_OPTION_SEND_CONFIGURED_LABELS;
 
-        if (exporter_get_boolean(instance_name, "send automatic labels", CONFIG_BOOLEAN_NO))
+        if (exporter_get_boolean(instance_name, "send automatic labels",
+                                (tmp_ci_list->exporting_type == EXPORTING_CONNECTOR_TYPE_GRAPHITE ||
+                                 tmp_ci_list->exporting_type == EXPORTING_CONNECTOR_TYPE_GRAPHITE_HTTP) ?
+                                 CONFIG_BOOLEAN_YES:  CONFIG_BOOLEAN_NO))
```

Comment:
If you wanted to enable `send automatic labels` for Graphite by default - this code doesn't do, test it.

Reply:
I also had it enabled on global and I did not observe this. Thanks!